frontend/institution/remove_inst_mobile.html

```diff
@@ -0,0 +1,16 @@
+<md-dialog>
+  <gen-dialog title="{{removeInstCtrl.getTitle()}}" confirm-action="removeInstCtrl.removeInst">
+      <entity-showcase title="removeInstCtrl.institution.name" subtitle="removeInstCtrl.institution.institutional_email" 
+          avatar="removeInstCtrl.institution.photo_url"></entity-showcase>
+          <div ng-if="!removeInstCtrl.thereIsNoChild()" style="margin-top: 0.5em;">
+            <md-radio-group ng-model="removeInstCtrl.removeHierarchy" required>
+                <md-radio-button ng-value="true" class="md-primary">
```

Comment:
I think it would be less verbose to show the bellow text before the options
> **Quanto a toda a hierarquia abaixo dessa institutição, desejo:**

and the options would be

> **Manter**

> **Remover**